src/Examples/Flowsheets/HDA_flowsheet_with_distillation_solution_testing.ipynb

```diff
@@ -0,0 +1,1773 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tear_guesses = {\n",
+    "        \"flow_mol_phase_comp\": {\n",
+    "                (0, \"Vap\", \"benzene\"): 1e-5,\n",
+    "                (0, \"Vap\", \"toluene\"): 1e-5,\n",
+    "                (0, \"Vap\", \"hydrogen\"): 0.30,\n",
+    "                (0, \"Vap\", \"methane\"): 0.02,\n",
+    "                (0, \"Liq\", \"benzene\"): 1e-5,\n",
+    "                (0, \"Liq\", \"toluene\"): 0.30,\n",
+    "                (0, \"Liq\", \"hydrogen\"): 1e-5,\n",
+    "                (0, \"Liq\", \"methane\"): 1e-5},\n",
+    "        \"temperature\": {0: 303},\n",
+    "        \"pressure\": {0: 350000}}\n",
+    "\n",
+    "# Pass the tear_guess to the SD tool\n",
+    "seq.set_guesses_for(m.fs.H101.inlet, tear_guesses)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we need to tell the tool how to initialize a particular unit. We will be writing a python function which takes in a \"unit\" and calls the initialize method on that unit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def function(unit):\n",
+    "        unit.initialize(outlvl=idaeslog.INFO)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are now ready to initialize our flowsheet in a sequential mode. Note that we specifically set the iteration limit to be 5 as we are trying to use this tool only to get a good set of initial values such that IPOPT can then take over and solve this flowsheet for us. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "seq.run(m, function)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Inline Exercise:</b>\n",
+    "We have now initialized the flowsheet. Let us run the flowsheet in a simulation mode to look at the results. \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the solver object\n",
+    "solver = SolverFactory('ipopt')\n",
+    "solver.options = {'tol': 1e-6, 'max_iter': 5000}\n",
+    "\n",
+    "# Solve the model\n",
+    "results = solver.solve(m, tee=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check solver solve status\n",
+    "from pyomo.environ import TerminationCondition\n",
+    "assert results.solver.termination_condition == TerminationCondition.optimal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add distillation column \n",
+    "\n",
+    "This completes the simulation of the flowsheet constructed earlier. We will now \n",
```

Comment:
I would add a little more explanation here, mainly a call back to earlier and say we will now add the distillation column. You can then go into the the specific steps required.

Reply:
I have modified the text to the following:
"As mentioned earlier, the SequentialDecomposition tool currently does not support the distillation column model. Thus, we have not included the distillation column in the flowsheet. Now that we have a converged flowsheet, we will add the distillation column and simulate the entire flowsheet. In the following, we will..."

Please let me know if you want me to include any other point.